### DISCUSSION TITLE
Hey can I setup hotkey to toggle from turning off dark reader to use system color scheme (instead of toggling dark reader on)?i mean when pressing(alt+shift+D)

### DISCUSSION THREAD

**Discussion by ghost** [darkreader/darkreader#8311] [Q&A]
Hey can I setup hotkey to toggle from **turning off dark reader** to **use system color scheme** (instead of toggling dark reader on)?i mean when pressing(alt+shift+D)

**Comment by Gusted** [comment]
We currently don't have such shortcut available.
